plugins/GSdx/GSDevice11.cpp

```diff
@@ -30,13 +30,21 @@ GSDevice11::GSDevice11()
 {
 	memset(&m_state, 0, sizeof(m_state));
 	memset(&m_vs_cb_cache, 0, sizeof(m_vs_cb_cache));
+	memset(&m_gs_cb_cache, 0, sizeof(m_gs_cb_cache));
 	memset(&m_ps_cb_cache, 0, sizeof(m_ps_cb_cache));
 
 	FXAA_Compiled = false;
 	ExShader_Compiled = false;
 	
 	m_state.topology = D3D11_PRIMITIVE_TOPOLOGY_UNDEFINED;
 	m_state.bf = -1;
+
+	if (theApp.GetConfigB("UserHacks")) {
+		UserHacks_unscale_pt_ln = theApp.GetConfigB("UserHacks_unscale_point_line");
+	}
+	else {
```

Comment:
:cry: 

I was close to use clang. But I need to import 2 patches from the old Gabest branch (openCL). And there is also the issue with the VR stuff.